src/main/java/frc/robot/RobotMap.java

```diff
@@ -71,6 +73,9 @@
 	public static int spoolLiftRightMotor = 11;
 	public static int robotLiftLeftMotorPort = 9; 
 
+	public static int flywheelForwardPort = 5;
+	public static int flywheelReversePort = 4;
```

Comment:
Unused values, I believe, now that we are using only one motor.

Reply:
Yep, looks like it was leftover from older commits